### PR TITLE
switch between ruby and haml formatters

### PR DESCRIPTION
**Pull request by Petercopter**
Partial fix for https://github.com/ruby-syntax-tree/vscode-syntax-tree/issues/45

### CODE REVIEW

**Comment by Petercopter** [comment]
@kddnewton Here's what I came up with for switching between the different formatters. Let me know what you think.

File: lib/syntax_tree/language_server.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@ def format(source)
             character: 0
           }
         },
-        newText: SyntaxTree.format(source, print_width)
+        newText: file_extension == "haml" ? SyntaxTree::Haml.format(source, print_width) : SyntaxTree.format(source, print_width)
```
Comment (kddnewton) [inline lib/syntax_tree/language_server.rb:101]: This can be `SyntaxTree::HANDLERS[".#{file_extension}"].format(source, print_width)`

**Review by xeger — APPROVED** [review]

**Comment by kddnewton** [comment]
Looks great!

**Comment by Petercopter** [comment]
@kddnewton Thanks for letting me get involved! 👍 🎉

**Pull request merged**